src/bp-core/admin/bp-core-admin-tools.php

```diff
@@ -871,6 +871,246 @@ function bp_core_admin_debug_information( $debug_info = array() ) {
 		)
 	);
 
+	$debug_info['buddypress-constants'] = array(
+		'label'       => __( 'BuddyPress Constants', 'buddypress' ),
+		'description' => __( 'These settings can alter where and how parts of BuddyPress are loaded or works.', 'buddypress' ),
+		'fields'      => array(
+			'BP_VERSION' => array(
+				'label' => 'BP_VERSION',
+				'value' => BP_VERSION,
+			),
+			'BP_DB_VERSION' => array(
+				'label' => 'BP_DB_VERSION',
+				'value' => BP_DB_VERSION,
+			),
+			'BP_REQUIRED_PHP_VERSION' => array(
+				'label' => 'BP_REQUIRED_PHP_VERSION',
+				'value' => BP_REQUIRED_PHP_VERSION,
+			),
+			'BP_PLUGIN_DIR' => array(
+				'label' => 'BP_PLUGIN_DIR',
+				'value' => BP_PLUGIN_DIR,
+			),
+			'BP_PLUGIN_URL' => array(
+				'label' => 'BP_PLUGIN_URL',
+				'value' => BP_PLUGIN_URL,
+			),
+			'BP_LOAD_SOURCE' => array(
+				'label' => 'BP_LOAD_SOURCE',
+				'value' => defined( 'BP_LOAD_SOURCE' ) ? BP_LOAD_SOURCE : __( 'Undefined', 'buddypress' ),
+				'debug' => defined( 'BP_LOAD_SOURCE' ) ? BP_LOAD_SOURCE : 'undefined',
+			),
+			'BP_IGNORE_DEPRECATED' => array(
+				'label' => 'BP_IGNORE_DEPRECATED',
+				'value' => defined( 'BP_IGNORE_DEPRECATED' ) && BP_IGNORE_DEPRECATED ? __( 'Enabled', 'buddypress' ) : __( 'Disabled', 'buddypress' ),
+			),
+			'BP_LOAD_DEPRECATED' => array(
+				'label' => 'BP_LOAD_DEPRECATED',
+				'value' => defined( 'BP_LOAD_DEPRECATED' ) && BP_LOAD_DEPRECATED ? __( 'Enabled', 'buddypress' ) : __( 'Disabled', 'buddypress' ),
+			),
+			'BP_FORUMS_PARENT_FORUM_ID' => array(
```

Comment:
This one might be to deprecate I need to give it a check 🤔

Reply:
Inline comment informs:
`// Legacy forum constant - supported for compatibility with bbPress 2.`

I'm not sure it's needed anymore as the legacy forum has been retired a very long time ago, but I'm sure we shouldn't include this constant into the panel.

Reply:
Since it is not deprecated yet, we will keep but without the `deprecated` label.